float32[] goal_state
float32 goal_radius
float32 time_limit
float32 probability_success_threshold